das/s3_das.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package das
+
+import (
+	"bytes"
+	"context"
+	"encoding/base32"
+	"errors"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/offchainlabs/nitro/arbstate"
+	"github.com/offchainlabs/nitro/blsSignatures"
+
+	flag "github.com/spf13/pflag"
+)
+
+var s3DasMutex sync.Mutex
```

Comment:
I don't think this mutex is necessary here. (In fact I should take it out of the other local disk DAS too, I had only put it in because I was trying to track down some race conditions that turned out to be elsewhere.)

Reply:
Remove mutex from s3 and local das